lib/paperclip/media_type_spoof_detector.rb

```diff
@@ -38,11 +38,11 @@ def supplied_type_mismatch?
     end
 
     def calculated_type_mismatch?
-      !media_types_from_name.include?(calculated_media_type)
+      media_types_from_name.none? {|media_type| calculated_media_types.include? media_type}
```

Comment:
Line is too long. [91/80]<br>Space between { and | missing.<br>Space missing inside }.